mkdocs.yml

```diff
@@ -130,6 +131,21 @@ plugins:
   - mkdocs-video:
       is_video: True
       video_autoplay: True
+  - gallery:
+      conf_script: docs/gallery_conf.py
+      examples_dirs: [
+        docs/examples,
+        docs/examples/library,
+        docs/examples/plugin
+      ]
+      gallery_dirs: [
+        docs/generated/gallery,
+        docs/generated/gallery/library,
+        docs/generated/gallery/plugin
+      ]
+      filename_pattern: /*.py  # which scripts will be executed for the docs
+      ignore_pattern: /mesh_headlight*
```

Comment:
This line can be removed when issue https://github.com/napari-threedee/napari-threedee/issues/103 is resolved